experiment/revert-bump.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
I personally prefer the readability of `set -o errexit` over the brevity of `set -e`

Reply:
done 🤷‍♂